lib/route.js

```diff
@@ -17,19 +18,25 @@ var Joi = require('joi');
 /**
  * Setup routes for a resource, given a name and model.
  *
+ * @param {Object} harvester
  * @param {String} name
  * @param {Object} model
+ * @param {Object} schema
+ * @param {Object} routeOptions
  */
-function route(name, model, schema, routeOptions) {
+function route(harvester, name, model, schema, routeOptions) {
```

Comment:
Using harvester rather than this makes the code way more straightforward. Good job there.